src/ripple/app/ledger/Ledger.cpp

```diff
@@ -942,6 +942,8 @@ static bool saveValidatedLedger (
                 JLOG (j.warn())
                     << "Transaction in ledger " << seq
                     << " affects no accounts";
```

Comment:
True.  I've seen other JSON dumps go on their own line in some other spots of the log, but I'm not sure if that is universal.